### PR TITLE
builtin: fix termux prints

### PR DESCRIPTION
**Pull request by treffner**
Fix for missing v output on termux/android.
#9662 .

This is a follow up to #9829 .
This should fix normal print(), eprint() and eprintln().

### CODE REVIEW

File: vlib/builtin/builtin.c.v [hunk 1]
```diff
@@ -119,13 +119,12 @@ pub fn eprintln(s string) {
...
 				C.fprintf(C.stderr, c'%.*s\n', s.len, s.str)
 			}
-		} $else {
-			if s.str == 0 {
-				_ = C.write(2, c'eprintln(NIL)\n', 14)
-			} else {
-				_ = C.write(2, s.str, s.len)
-				_ = C.write(2, c'\n', 1)
-			}
+		}
```
Comment (spytheman) [inline vlib/builtin/builtin.c.v:122]: just return early, nested comptime code is hard to follow

Comment (treffner) [reply vlib/builtin/builtin.c.v:122]: The commit removes one layer of nesting.
We output the special android case that (I heard that is for android catlog, but I cannot confirm this because vab does not find my android tools) and we also output the normal case for the termux terminal.
